packages/rum/package.json

```diff
@@ -51,11 +51,11 @@
   "bundlesize": [
     {
       "path": "./dist/bundles/elastic-apm-rum*.min.js",
-      "maxSize": "16 kB"
+      "maxSize": "20 kB"
```

Comment:
Pretty hesitant to increase it by a lot, Can we keep it to `18.5kB` for this one and `opentracing ` as `20kB`?

Reply:
I'd like to understand more about these checks at the test level: are they critical enough to have the limits hardcoded in the package.json file? How often do those limits change (== how often would the tests fail after resizing the bundle)?

Reply:
It depends on features/bug fixes that goes inside the agent code that is being shipped to the users so if we are increasing it by a lot we would have to catch it during our tests and make sure the change is intended as per the plan. 

If we did merge a PR with the increased code size, then the RUM team would need to work on the optimization phase if its worth it or increase the maxSize to fix the test failures. 

Does that make sense?

Reply:
What about defining acceptable thresholds for increments? Something like increasing the bundle a small % of the size of the released one 🤔

Reply:
We already have this check https://github.com/elastic/apm-agent-rum-js/pull/863#issuecomment-669099572 which shows us the thresholds, so i guess we are good here.